src/tests.c

```diff
@@ -901,6 +901,27 @@ void ge_equals_ge(const secp256k1_ge_t *a, const secp256k1_ge_t *b) {
     CHECK(secp256k1_fe_equal_var(&b->y, &b->y));
 }
 
+int gej_equals_gej(const secp256k1_gej_t *a, const secp256k1_gej_t *b) {
```

Comment:
Perhaps add a comment that this compares the actual representation of the gej (including the Z) coordinate, and not just its geometric meaning?